### PR TITLE
matlab: add page

### PR DESCRIPTION
**Pull request by jpiamjariyakul**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/matlab.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# MATLAB
```
Comment (navarroaxel) [inline pages/common/matlab.md:1]: This should be the name of the command:
```suggestion
# matlab
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Nice @jpiamjariyakul, just 1 comment.

**Review by jpiamjariyakul — COMMENTED** [review]
Thanks @navarroaxel, I've made the edit per suggested

**Review by navarroaxel — APPROVED** [review]

File: pages/common/matlab.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# matlab
+
+> Numerical computation environment by MathWorks.
+> More information: <https://uk.mathworks.com/help/matlab/>.
+
+- Run without splash screen during startup:
+
+`matlab -nosplash`
```
Comment (sbrl) [inline pages/common/matlab.md:8]: Missing dash? Not a matlab user, so I'm just checking to make sure.

File: pages/common/matlab.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Execute a MATLAB statement:
+
+`matlab -r {{matlab_statement}}`
```
Comment (sbrl) [inline pages/common/matlab.md:12]: Perhaps quotes would be a good idea here.
```suggestion
`matlab -r "{{matlab_statement}}"`
```
Also, is there a long-form of the `-r` argument here? We generally refer long-form arguments to aid clarity.

**Review by sbrl — COMMENTED** [review]
Thanks for the new page, @jpiamjariyakul ! I've left some comments below for you to review.

File: pages/common/matlab.md (hunk 2 above)
Comment (jpiamjariyakul) [reply pages/common/matlab.md:8]: Yep that's correct, MATLAB only uses one dash even in linux (it's a weird one, I know...)

**Review by jpiamjariyakul — COMMENTED** [review]
Applied the suggested changes! Also see the comment regarding MATLAB having only one dash, it's an interesting point.

**Review by owenvoke — APPROVED** [review]

File: pages/common/matlab.md (hunk 2 above)
Comment (sbrl) [reply pages/common/matlab.md:8]: Oh, that's odd!

**Review by sbrl — APPROVED** [review]

**Comment by sbrl** [comment]
Thanks, @jpiamjariyakul :smiley_cat:

**Pull request merged**